packages/teleport/src/Databases/Databases.story.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import Databases from './Databases';
+import { databases } from './fixtures';
+
+export function Loaded() {
+  return render();
```

Comment:
This file contains only one story, so we don't need to use a helper.

```
export const Loaded = () => <Databases databases={databases} rest of props.../>
```

You are probably thinking of next issue, but @alex-kovoy recently mentioned a shorter/simpler way of rendering same story with different props, so keep in mind (for the next issue) that this will be the new way we will define stories: https://storybook.js.org/docs/react/writing-stories/args

more about bind: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind